src/core.js

```diff
@@ -8,9 +8,12 @@ import exportQUnit from "./export";
 
 import config from "./core/config";
 import { defined, extend, objectType, is, now } from "./core/utilities";
-import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
+import { on, emit, registerLoggingCallbacks,
+	runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
 
+import jsReporters from "js-reporters";
```

Comment:
Can we import individual modules directly? Importing the whole thing is likely to include all the adapters in the QUnit bundle, which we don't want.